apollo-portal/src/main/java/com/ctrip/framework/apollo/portal/component/txtresolver/PropertyResolver.java

```diff
@@ -72,24 +73,24 @@ public ItemChangeSets resolve(long namespaceId, String configText, List<ItemDTO>
     return changeSets;
   }
 
-  private boolean isHasRepeatKey(String[] newItems) {
+  private boolean isHasRepeatKey(String[] newItems,@NotNull Set<String> repeatKeys) {
```

Comment:
```suggestion
  private boolean isHasRepeatKey(String[] newItems, @NotNull Set<String> repeatKeys) {
```

BTW, this is a private method, so maybe `@NotNull` is not necessary?